packages/cli/test/integration/project.api.test.ts

```diff
@@ -432,6 +433,32 @@ describe('POST /projects/', () => {
 
 		expect(await Container.get(ProjectRepository).count({ where: { type: 'team' } })).toBe(2);
 	});
+
+	test('should respect the quota when trying to create multiple projects in parallel (no race conditions)', async () => {
```

Comment:
Out of curiosity I added this test in master and it passes anyway?

```
  PATCH /projects/:projectId
    ✓ should respect the quota when trying to create multiple projects in parallel (no race conditions) (2 ms)
    ✓ should update a team project name (14 ms)
    ✓ should not allow viewers to edit team project name (14 ms)
    ✓ should not allow owners to edit personal project name (21 ms)
```

Reply:
Do you have `DB_SQLITE_POOL_SIZE` set to something higher than 0?

Reply:
Personally I don't really mind how we do this as long as we pick one way:

1. conditionally pass the test (what I've done here)
2. conditionally not scheduling it (what I will do now)
3. conditionally skipping the test (is a bit harder and requires extracting the test function, giving it a name and then calling `it(testName, testFn)` or `it.skip(testName, testFn)` conditionally, which may throw of some eslint rules)

If you prefer any of that let me know, I don't think we should bikeshed this but just pick one and move on 😄

Reply:
Cases like this should be vanishingly rare, no strong feelings on this so up to you.